### DISCUSSION TITLE
extensions.filters.network.http_connection_manager.v3.HttpConnectionManager's drain_timeout option is allow only in Http2?

### DISCUSSION THREAD

**Discussion by Livenow14** [istio/istio#49201] [Q&A] [answered]
[docs](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto)
I looked at the docs above, but the description of drain_timeout says it only works on HTTP2, so I have a question.

Q. Doesn't this setting work on HTTP 1.1?

**Comment by howardjohn** [comment, marked as answer]
No, its an HTTP2 only field. delayed_close_timeout is fairly similar for HTTP/1.1 though.

Note all of this is Envoy specific and not configurable in Istio.

**Reply by Livenow14** [reply]
Omg, I barking up the wrong tree. thanks you so much with your answer.
